.buildconfig.yml

```diff
@@ -0,0 +1,254 @@
+    description: 'Engine implementation based on the system WebView.'
+    publish: true
+  browser-engine-servo:
+    path: components/browser/engine-servo
+    description: 'Engine implementation based on the Servo Browser Engine.'
+    publish: true
+  browser-errorpages:
+    path: components/browser/errorpages
+    description: 'Responsive browser error pages for Android apps.'
+    publish: true
+  browser-menu:
+    path: components/browser/menu
+    description: 'A customizable menu for browsers.'
+    publish: true
+  browser-search:
+    path: components/browser/search
+    description: 'Search plugins and companion code to load, parse and use them.'
+    publish: true
+  browser-session:
+    path: components/browser/session
+    description: 'An abstract layer hiding the actual browser engine implementation.'
+    publish: true
+  browser-storage-memory:
+    path: components/browser/storage-memory
+    description: 'A memory-backed implementation of core data storage.'
+    publish: true
+  browser-storage-sync:
+    path: components/browser/storage-sync
+    description: 'A syncable, Rust Places-backed implementation of core data storage.'
+    publish: true
+  browser-tabstray:
+    path: components/browser/tabstray
+    description: 'A tabs tray component for browsers.'
+    publish: true
+  browser-toolbar:
+    path: components/browser/toolbar
+    description: 'A customizable toolbar for browsers.'
+    publish: true
+  ui-autocomplete:
+    path: components/ui/autocomplete
+    description: 'A set of components to provide autocomplete functionality.'
+    publish: true
+  ui-colors:
+    path: components/ui/colors
+    description: 'The standard set of Photon colors.'
+    publish: true
+  ui-fonts:
+    path: components/ui/fonts
+    description: 'Convenience accessor for fonts used by Mozilla.'
+    publish: true
+  ui-icons:
+    path: components/ui/icons
+    description: 'A collection of often used browser icons.'
+    publish: true
+  ui-progress:
+    path: components/ui/progress
+    description: 'An animated progress bar following the Photon Design System.'
+    publish: true
+  ui-tabcounter:
+    path: components/ui/tabcounter
+    description: 'A tab counter for browsers.'
+    publish: true
+  service-firefox-accounts:
+    path: components/service/firefox-accounts
+    description: 'A library for integrating with Firefox Accounts.'
+    publish: true
+  service-sync-logins:
+    path: components/service/sync-logins
+    description: 'A library for integrating with Firefox Sync - Logins.'
+    publish: true
+  service-fretboard:
+    path: components/service/fretboard
+    description: 'An Android framework for segmenting users in order to run A/B tests and rollout features gradually.'
+    publish: true
+  service-glean:
+    path: components/service/glean
+    description: 'A client-side telemetry SDK for collecting metrics and sending them to the Mozilla telemetry service'
+    publish: true
```

Comment:
@pocmo does this mean the library will be officially published? Is this retaining the same behaviour as the old code?

Reply:
> Define "officially". :)

Good point :) Together with other libraries from android-components.

> But yes, it's getting published since the beginning :)
> https://maven.mozilla.org/?prefix=maven2/org/mozilla/components/service-glean/

Perfect then!